### PR TITLE
Add the second step of CC

### PR DESCRIPTION
**Pull request by evgenytalanin-intel**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: .ci/azure/linux_conditional_compilation.yml [hunk 1]
```diff
@@ -83,16 +96,37 @@ jobs:
         $(REPO_DIR)
       workingDirectory: $(BUILD_DIR)
 
-  - script: ninja
+  - script: |
+      ninja
+      cmake --build . --target sea_itt_lib
```
Comment (ilyachur) [inline .ci/azure/linux_conditional_compilation.yml:101]: You need build all as before and after that build `sea_itt_lib` separatelly

**Comment by ilyachur** [comment]
@evgenytalanin-intel Do you have a plan to finalize this PR?

**Comment by ilya-lavrenov** [comment]
can we proceed with this? looks quite useful

**Comment by evgenytalanin-intel** [comment]
> can we proceed with this? looks quite useful

We are about to add Windows as well, and then we will merge it

**Review by SDxKeeper — APPROVED** [review]

File: .ci/azure/linux_conditional_compilation.yml [hunk 2]
```diff
@@ -83,16 +96,37 @@ jobs:
...
+        -DENABLE_FASTER_BUILD=ON
+        -DENABLE_PROFILING_ITT=OFF
+        -DSELECTIVE_BUILD=ON
+        -DSELECTIVE_BUILD_STAT=$(BUILD_DIR)/*.csv
+        $(REPO_DIR)
+      workingDirectory: $(BUILD_DIR_2)
+
+  - script: cmake --build .
+    workingDirectory: $(BUILD_DIR_2)
+    displayName: 'Build LinCC ON'
```
Comment (azhogov) [inline .ci/azure/linux_conditional_compilation.yml:128]: what is ON?

**Review by azhogov — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
PIIIIING!

**Pull request closed without merging**